translations/en.json

```diff
@@ -768,7 +768,8 @@
                 "activate": "Activate"
             },
             "script": {
-                "execute": "Execute"
+                "execute": "Execute",
+                "last_action": "Last Action"
```

Comment:
This is the wrong place to add translations. it has to be in `src` dir.

Also, we shouldn't add translations for more info dialogs under the card key. Those are two different things.

Reply:
damn, right, fixed the wrong translation file.

But what translate key should be used here? All ~40 translation in the "more informaion dialogs" in found are located in "ui.card." translation context. Any suggestion?

Reply:
Let's create a new `more_info_control` under `dialogs`

Reply:
And then keyed off by domain

Reply:
kk, should I also migrate the ~40 old ui.card.* usages inside the dialogs/more-info directory (might mark the translation as new in translate tool?!)? Or should it be part of another PR?